src/project/qml/MuseScore/Project/internal/Export/MeiSettingsPage.qml

```diff
@@ -39,4 +39,18 @@ ExportSettingsPage {
             root.model.meiExportLayout = !checked
         }
     }
+
+    CheckBox {
+        width: parent.width
+        text: qsTrc("project/export", "Use MuseScore Ids")
```

Comment:
Maybe, in the UI, call it like this:
```suggestion
        text: qsTrc("project/export", "Use MuseScore element IDs")
```

(In the code, the spelling `Id` is fine, but for the UI I'd use `ID`)